packages/sdk-js/src/sdk-client.ts

```diff
@@ -0,0 +1,207 @@
+
+import { FeeMarketEIP1559Transaction } from "@ethereumjs/tx";
+import elliptic from 'elliptic';
+
+export class TurnkeySDKClient extends TurnkeySDKClientBase {
+  constructor(config: TurnkeySDKClientConfig) {
+    super(config);
+  }
+
+  // RPC URL to Send Transactions?
+
+  // Transaction Helpers
+  // UserConfirmation
+  signTransactionObject = async (params: { signWith: string, tx: EthereumTransaction }): Promise<SdkApiTypes.TSignTransactionResponse> => {
+    const encodedTransaction = FeeMarketEIP1559Transaction.fromTxData(params.tx);
+    const formattedEncodedTransaction = bytesToHex(encodedTransaction.getMessageToSign()).slice(2);
+
+    return await this.signTransaction({
+      signWith: params.signWith,
+      unsignedTransaction: formattedEncodedTransaction,
+      type: "TRANSACTION_TYPE_ETHEREUM"
+    })
+  }
+
+  // Wallet Helpers
+  // UserConfirmation
+  createWalletWithAccount = async (params: { walletName: string, chain: string; }): Promise<SdkApiTypes.TCreateWalletResponse> => {
+    if (params.chain === "ethereum") {
+      return await this.createWallet({
+        walletName: params.walletName,
+        accounts: [{
+          curve: "CURVE_SECP256K1",
+          pathFormat: "PATH_FORMAT_BIP32",
+          path: "m/44'/60'/0'/0/0",
+          addressFormat: "ADDRESS_FORMAT_ETHEREUM"
+        }]
+      })
+    } else {
+      return await this.createWallet({
+        walletName: params.walletName,
+        accounts: [{
+          curve: "CURVE_SECP256K1",
+          pathFormat: "PATH_FORMAT_BIP32",
+          path: "m/44'/60'/0'/0/0",
+          addressFormat: "ADDRESS_FORMAT_ETHEREUM"
+        }]
+      })
+    }
+  }
+
+  // UserConfirmation
+  createNextWalletAccount = async (params: { walletId: string }): Promise<SdkApiTypes.TCreateWalletAccountsResponse> => {
+    const walletAccounts = await this.getWalletAccounts({ walletId: params.walletId });
+    const lastAccount = walletAccounts.accounts[walletAccounts.accounts.length - 1]!;
+    const lastAccountPath = lastAccount.path.split("/");
+    const lastAccountPathIndex = lastAccountPath[3]!.replace(/[^0-9]/g, '');
+    const nextPathIndex = Number(lastAccountPathIndex) + 1;
+    lastAccountPath[3] = `${nextPathIndex}'`;
+    const nextAccountPath = lastAccountPath.join("/");
+    return this.createWalletAccounts({
+      walletId: params.walletId,
+      accounts: [{
+        curve: lastAccount.curve,
+        pathFormat: lastAccount.pathFormat,
+        addressFormat: lastAccount.addressFormat,
+        path: nextAccountPath
+      }]
+    })
+  }
+
+  // User Auth
+  // API
+  createUserAccount = async (email: string): Promise<SdkApiTypes.TCreateSubOrganizationResponse> => {
+    const challenge = generateRandomBuffer();
+    const authenticatorUserId = generateRandomBuffer();
+
+    const attestation = await getWebAuthnAttestation({
+      publicKey: {
+        rp: {
+          id: "localhost",
+          name: "Demo Passkey Wallet"
```

Comment:
we can probably update this to use a generic name, like `Default App Name` or something